user-interface/src/main/java/life/qbic/datamanager/views/register/RegistrationOrcIdMain.java

```diff
@@ -99,13 +99,13 @@ private void handleRegistrationFailure(List<RuntimeException> exceptionList) {
     if (exceptionList.isEmpty()) {
       return;
     }
-    if (exceptionList.contains(UserExistsException.class)) {
+    if (exceptionList.stream().anyMatch(e -> e instanceof UserExistsException)) {
```

Comment:
why these changes?

Reply:
In a nutshell the contains method does not work for instance based comparison in generics. Since we're only interested in the type of exception thrown, this had to be adapted.